www/src/css/branding.css

```diff
@@ -0,0 +1,24 @@
+/** @define b-branding-header; weak */
```

Comment:
What about injecting the text and link color attributes in that stylesheet. I'd like to see the logic in place, and get the defaults from the model rather than in place here. Is that really all doable at runtime ?!